pages/data/index.vue

```diff
@@ -522,6 +536,47 @@ export default {
         })
     },
 
+    /**
+     * Get organ details from discover api
+     * @param {Object}
+     * @returns {Object}
+     */
+    getOrganDetails: function(organ) {
+      const organType = pathOr('', ['fields', 'name'], organ)
+      return this.$axios
+        .get(
+          `${
+            process.env.discover_api_host
+          }/search/datasets?query=${organType.toLowerCase()}&limit=100`
```

Comment:
This could potentially be `limit=1` which would, in theory, be less intensive and respond faster.

Reply:
I'll test that out. Yeah, it is intensive only because of the pagination, I agree it could be in a separate endpoint down the line.